### PR TITLE
Make users cache expiration delay configurable

### PR DESCRIPTION
**Pull request by gwenneg**

### CODE REVIEW

File: connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/Environment.java
Comment (gwenneg) [inline connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/Environment.java:1]: @MikelAlejoBR This class doesn't seem to be needed. Could you please confirm that?

File: connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/config/EmailConnectorConfig.java [hunk 1]
```diff
@@ -93,6 +94,9 @@ public class EmailConnectorConfig extends ConnectorConfig {
     @ConfigProperty(name = RBAC_URL)
     String rbacURL;
 
+    @ConfigProperty(name = USER_PROVIDER_CACHE_EXPIRE_AFTER_WRITE, defaultValue = "600")
```
Comment (gwenneg) [inline connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/config/EmailConnectorConfig.java:97]: We're currently caching users for 10 minutes in prod. Let's do the same in the connector.

**Review by MikelAlejoBR — APPROVED** [review]
LGTM!

File: connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/Environment.java
Comment (MikelAlejoBR) [reply connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/Environment.java:1]: This is right, yes. We can delete that class.

**Pull request merged**